tests/test_metadata_rules.py

```
@@ -0,0 +1,573 @@
+            "labels": ["kubevirt-console"],
+        },
+        {
+            "pattern": "test-pytest-cnv-*-storage-*",
+            "team": "storage",
+            "tier": "t2",
+            "labels": ["pytest"],
+        },
+        {"pattern": "dev-*-rnetser", "labels": ["dev", "personal"]},
+        {"pattern": "*-gating", "labels": ["gating"]},
+        {"pattern": "verify-*-wrapper", "team": "ci", "labels": ["wrapper", "verify"]},
+    ]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)